modules/docker-build/variables.tf

```diff
@@ -0,0 +1,18 @@
+variable "working_dir" {}
+
+variable "image_name" {}
+
+variable "short_name" {}
+
+variable "image_tag" {}
+
+variable "docker_registry" {}
+
+variable "dockerfile" {
+  default = "Dockerfile"
+}
+
+variable "depends_on" {
```

Comment:
this variable gets set from top-level modules (e.g. `account/main.tf`), but is not used inside the module `docker-build`

Reply:
`depends_on` does not support interpolations. This just creates an implicit dependency that one module completes before the other.